backend/src/main/java/de/bund/digitalservice/ris/caselaw/adapter/database/jpa/PostgresFieldOfLawRepositoryImpl.java

```diff
@@ -72,119 +76,57 @@ public Slice<FieldOfLaw> findAllByOrderByIdentifierAsc(Pageable pageable) {
 
   @Override
   @Transactional
-  public List<FieldOfLaw> findBySearchTerms(String[] searchTerms) {
-    if (searchTerms == null || searchTerms.length == 0) {
-      return Collections.emptyList();
+  public List<FieldOfLaw> findByCombinedCriteria(
+      String identifier, String[] descriptionSearchTerms, String[] normSearchTerms) {
+    CriteriaBuilder cb = entityManager.getCriteriaBuilder();
+    CriteriaQuery<FieldOfLawDTO> cq = cb.createQuery(FieldOfLawDTO.class);
+    Root<FieldOfLawDTO> fieldOfLawRoot = cq.from(FieldOfLawDTO.class);
+    fieldOfLawRoot.fetch("norms", JoinType.LEFT);
+    ArrayList<Predicate> predicates = new ArrayList<>();
+
+    Predicate notationPredicate =
+        cb.equal(fieldOfLawRoot.get("notation"), cb.literal(Notation.NEW));
```

Comment:
question: Do we ever search by `Notation.NEW` or do we have them just for legacy doc units?

Reply:
I assume you mean `Notation.OLD`, and that's exactly right, those are deprecated and just exist in the table for the old doc units :)